gc/base/MemoryManager.cpp

```diff
@@ -95,6 +95,12 @@ MM_MemoryManager::createVirtualMemoryForHeap(MM_EnvironmentBase* env, MM_MemoryH
 		}
 	}
 
+#if defined(LINUX)
+	if(extensions->isVLHGC() && extensions->indexableObjectModel.isDoubleMappingEnabled()) {
```

Comment:
I think we need to define `bool isDoubleMappingEnabled()` in [ArrayletObjectModel.hpp](https://github.com/eclipse/omr/blob/f8fc704032f95cbc0f337eb6be54f2b06e58a773/example/glue/ArrayletObjectModel.hpp) 

```c++
	MMINLINE bool
	isDoubleMappingEnabled()
	{
		return false;
	}
```